crates/arrow_sql_gen/tests/clickhouse_test.rs

```diff
@@ -0,0 +1,73 @@
+use arrow_sql_gen::clickhouse::block_to_arrow;
+
+#[cfg(feature = "clickhouse")]
+#[test]
+fn test_block_to_arrow() {
+    use clickhouse_rs::{types::Decimal, Block};
+
+    let block = Block::new()
+        .add_column("int_8", vec![1_i8, 2, 4])
+        .add_column("int_16", vec![1_i16, 2, 4])
+        .add_column("int_32", vec![1_i32, 2, 4])
+        .add_column("int_64", vec![1_i64, 2, 4])
+        .add_column("uint_8", vec![1_u8, 2, 4])
+        .add_column("uint_16", vec![1_u16, 2, 4])
+        .add_column("uint_32", vec![1_u32, 2, 4])
+        .add_column("uint_64", vec![1_u64, 2, 4])
+        .add_column("float_32", vec![1.0_f32, 2.0, 4.0])
+        .add_column("float_64", vec![1.0_f64, 2.0, 4.0])
+        .add_column("string", vec!["a", "b", "c"])
+        .add_column(
+            "uuid",
+            vec![
+                uuid::Uuid::default(),
+                uuid::Uuid::default(),
+                uuid::Uuid::default(),
+            ],
+        )
+        .add_column("nullable", vec![Some(1), None, Some(3)])
```

Comment:
Just as a curiosity - does clickhouse detect type internally based on first value? So basically column name does not matter here?

Nit: Better to name`nullable(int_32)`